website/docs/r/cluster.html.markdown

```diff
@@ -88,6 +88,9 @@ The following arguments are supported:
 * `eks_config` - (Optional) The Amazon eks configuration for `eks` Clusters. Conflicts with `aks_config`, `gke_config` and `rke_config` (list maxitems:1)
 * `gke_config` - (Optional) The Google gke configuration for `gke` Clusters. Conflicts with `aks_config`, `eks_config` and `rke_config` (list maxitems:1)
 * `description` - (Optional) The description for Cluster (string)
+* `cluster_auth_endpoint` - (Optional/Computed) Enabling the authorized cluster endpoint allows direct communication with the cluster, bypassing the Rancher API proxy. (list maxitems:1)
```

Comment:
Maybe link to Rancher documentation talking about this feature.

Reply:
Good point. Added link to Rancher documentation for `cluster_auth_endpoint` and `default_pod_security_policy_template_id` features.